wordpress-mobile/WordPress-iOS
# Auto-search Domains suggestion when entering Domains selection screen

Fixes #10976 

When entering domains we want to search based off the title in the Basic Information screen. This change first checks if the textfield currently has any text and is not empty. If so, it uses that for a search query. Then, it checks whether the basic information title is set and not empty.

If none of these hold true, it simply returns.

To test:

1. Go to Enhanced Site Creation and start creating a new site
2. Select any segment and type in any vertical
3. In Basic Information, type in any word for the title
4. Verify when clicking next that domain suggestions show up for that title
5. Go back to basic information and clear the title
6. Verify when clicking next that there are no domain suggestions
7. Verify that typing a domain in the search bar changes the domain suggestions

Update release notes:
